app/assets/javascripts/provider/admin/apiconfig/services/integration.js.coffee

```diff
@@ -8,6 +8,14 @@ $(document).on 'initialize', '#proxy', ->
   $("input.error").tipsy
     trigger: "focus"
 
+  # Warning for "catch-all" rules
+  setUpWarningForPattern = (pattern) ->
+    pattern.children('input').keyup (e) ->
+      warning = pattern.children('#catch-all-warning')
+      warning.css 'display', if e.currentTarget.value is '/' then 'block' else 'none'
```

Comment:
Instead of adding an inline style, you could add a class `hidden` or `show`

Reply:
Yes but I'm not sure it's worth it for such a tiny change. Is there any particular reason?

Reply:
I think we should avoid inline styles always as a good practice